Content.Server/Administration/BwoinkSystem.cs

```diff
@@ -60,15 +97,56 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
             foreach (var channel in targets)
                 RaiseNetworkEvent(msg, channel);
 
+            var sendsWebhook = _webhookUrl != string.Empty;
+            if (sendsWebhook)
+            {
+                var sawmill = IoCManager.Resolve<ILogManager>().GetSawmill("AHELP");
+
+                void LookupFinished(Task<LocatedPlayerData?> finishedTask)
+                {
+                    if (finishedTask.Result == null)
+                    {
+                        sawmill.Log(LogLevel.Error, $"Unable to find player for netuserid {msg.ChannelId} when sending discord webhook.");
+                        return;
+                    }
+
+                    var payload = new WebhookPayload()
```

Comment:
I would very much recommend you to set `allowed_mentions` here to avoid accidental pings. Then you probably don't have to bother with text escaping.

Reply:
i thought allowed_mentions defaults to being empty. just checked the api docs and... bruh
![grafik](https://user-images.githubusercontent.com/16618384/147087359-ce673496-f559-418a-bb4e-eba0acce48e3.png)
will fix